All_home_works/hw2/task5.py

```diff
@@ -0,0 +1,29 @@
+"""
+Some of the functions have a bit cumbersome behavior when we deal with
+positional and keyword arguments.
+
+Write a function that accept any iterable of unique values and then
+it behaves as range function:
+
+
+import string
+
+
+assert = custom_range(string.ascii_lowercase, 'g') == ['a', 'b', 'c', 'd', 'e', 'f']
+assert = custom_range(string.ascii_lowercase, 'g', 'p') == ['g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o']
+assert = custom_range(string.ascii_lowercase, 'p', 'g', -2) == ['p', 'n', 'l', 'j', 'h']
+
+"""
+
+
+def custom_range(iterable, start=None, stop=None, step=None):
```

Comment:
Will this function work with the following list:  `[1,2, 3, None, 4, 5]`?

Reply:
Ok, =). I understood. Thanks a lot. You help me to see bugs.